Source/RunActivity/Viewer3D/Cameras.cs

```diff
@@ -1798,6 +1810,8 @@ public override bool IsUnderground
                     return true;
                 if (TrackCameraLocation == WorldLocation.None) return false;
                 var elevationAtCameraTarget = Viewer.Tiles.GetElevation(TrackCameraLocation);
+//                if (TrackCameraLocation.Location.Y + TerrainAltitudeMargin < elevationAtCameraTarget)
+//                    Trace.TraceInformation("CameraY {0}, elevationAtCamera {1}", TrackCameraLocation.Location.Y, elevationAtCameraTarget);
```

Comment:
If you could remove all the commented out code, that'd be great. If you'd like to keep code like this for debugging in future, I'd recommend the `#if DEBUG_...` style that can be found in some other places.

Reply:
Done.